shared/src/metabase/shared/parameters/parameters.cljc

```diff
@@ -131,11 +154,13 @@
 (defn ^:export substitute_tags
   "Given the context of a text dashboard card, replace all template tags in the text with their corresponding values,
   formatted and escaped appropriately."
-  [text tag->param]
-  (let [tag->param #?(:clj tag->param
-                      :cljs (js->clj tag->param))
-        tag->normalized-param (reduce-kv (fn [acc tag param]
-                                           (assoc acc tag (normalize-parameter param)))
-                                         {}
-                                         tag->param)]
-     (str/replace text template-tag-regex (partial replacement tag->normalized-param))))
+  ([text tag->param]
+   (substitute_tags text tag->param "en"))
+  ([text tag->param locale]
+   (let [tag->param #?(:clj tag->param
+                       :cljs (js->clj tag->param))
+         tag->normalized-param (reduce-kv (fn [acc tag param]
+                                            (assoc acc tag (normalize-parameter param)))
```

Comment:
I know I asked about this in a previous review - I guess we don't have Clojure 1.11's `update-vals` yet?